analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ImplementISerializableCorrectly.CSharp10.cs

```diff
@@ -7,27 +7,33 @@ public class NonSerializedType
     {
     }
 
-    public struct SerializableStruct_NoAttribute : ISerializable // FN
+    public struct SerializableStruct_NoAttribute : ISerializable    // Noncompliant {{Update this implementation of 'ISerializable' to conform to the recommended serialization pattern.}}
+                                                                    // Secondary@-1 {{Add 'System.SerializableAttribute' attribute on 'SerializableStruct_NoAttribute' because it implements 'ISerializable'.}}
+                                                                    // Secondary@-2 {{Add a 'private' constructor 'SerializableStruct_NoAttribute(SerializationInfo, StreamingContext)'.}}
     {
-        private readonly NonSerializedType field = null; // FN, should be marked with [NonSerialized]
+        private readonly NonSerializedType field = null; // Should be marked with [NonSerialized]
```

Comment:
Why did you remove the FN? (Same for the other removals below).

Reply:
I was under the wrong impression that the secondary location messages tell to add it.